coral-component-colorinput/src/scripts/ColorInputSwatch.js

```diff
@@ -192,7 +196,13 @@ const ColorInputSwatch = Decorator(class extends BaseColorInputAbstractSubview(B
       button.remove();
     }
 
+    const label = this.querySelector('[handle="colorLabel"]');
+    if (label) {
+      label.remove();
+    }
+
     this.appendChild(this._elements.colorButton);
+    this.appendChild(this._elements.colorLabel);
```

Comment:
The `coral-tooltip` should target the `colorButton` element.
```suggestion
    this.appendChild(this._elements.colorLabel);
    
    this._elements.colorLabel.target = '#' + this._elements.colorButton.id;
```